metrics-jetty9/src/main/java/io/dropwizard/metrics5/jetty9/InstrumentedConnectionFactory.java

```diff
@@ -38,11 +53,13 @@ public Connection newConnection(Connector connector, EndPoint endPoint) {
             @Override
             public void onOpened(Connection connection) {
                 this.context = timer.time();
+                counterMaybe.ifPresent(Counter::inc);
```

Comment:
Could you add an integration test to `InstrumentedConnectionFactoryTest` to verify that the counter gets incremented when a connection opens, and get decremented when the client close it.

Reply:
Sure thing